web_poet/fields.py

```diff
@@ -91,27 +88,41 @@ def __set_name__(self, owner, name):
             getattr(owner, _FIELDS_INFO_ATTRIBUTE_WRITE)[name] = field_info
 
         def __get__(self, instance, owner=None):
+            if self.unbound_method is None:
+                processors = out
+                method = self._processed(self.original_method, processors, instance)
+                if cached:
+                    self.unbound_method = cached_method(method)
+                else:
+                    self.unbound_method = method
+
             return self.unbound_method(instance)
 
         @staticmethod
-        def _process(value):
-            for processor in out:
-                value = processor(value)
+        def _process(value, processors, instance):
+            for processor in processors:
+                sig = inspect.signature(processor)
```

Comment:
I think we should do it once, when the field is defined, not at every call to the field.